tools/ci/production-verify

```diff
@@ -56,6 +56,9 @@ if supervisorctl status | grep -vq RUNNING || supervisorctl status | sed 's/^.*u
     echo
     echo "DEBUG: printing Zulip server's tornado log:"
     cat /var/log/zulip/tornado.log
+    echo
+    echo "DEBUF: printing process_fts_updates log"
```

Comment:
`DEBUF` -> `DEBUG`

Reply:
This also might want to be its own commit.

Reply:
@alexmv In fairness, this was only in there whilst I was debugging my issues, so it isn't a necessity, but happy to add it in as a separate commit if you think its useful in the long run.